src/test/httpserver_tests.cpp

```diff
@@ -5,34 +5,55 @@
 #include <httpserver.h>
 #include <test/util/setup_common.h>
 
+#include <event2/http.h>
+
+#include <string>
+#include <string_view>
+
 #include <boost/test/unit_test.hpp>
 
 BOOST_FIXTURE_TEST_SUITE(httpserver_tests, BasicTestingSetup)
 
-BOOST_AUTO_TEST_CASE(test_query_parameters)
+std::optional<std::string> GetParameterFromURI(const std::string_view uri, const std::string& key){
+    auto uri_parsed{evhttp_uri_parse(uri.data())};
+    if (uri_parsed == nullptr) return std::nullopt;
+    auto param_value{GetQueryParameterFromUri(*uri_parsed, key)};
+    if (uri_parsed) evhttp_uri_free(uri_parsed);
```

Comment:
This will always be `true` because if `uri_parsed` is `nullptr` we would have returned 2 lines above.

Reply:
I'll change it. Thanks.